source/jobs/creating-jobs.txt

```diff
@@ -31,108 +35,126 @@ Before you Begin
+        for SQL Server.
 
-   2h. Click the :guilabel:`SSL` toggle switch to enable or disable SSL and select an SSL mode.
+        If you want to specify the JDBC URI manually, click the 
+        :guilabel:`Enter URI manually` toggle switch on the 
+        :guilabel:`Connect SourceDB` form. For details, see 
+        :ref:`rm-relational-database-connection-strings`.
 
-   2i. Click :guilabel:`Connect`.
+  .. step:: Enter the source MongoDB connection details 
 
-   .. note::
+      On the :guilabel:`Connect Destination DB` form, input your 
+      :guilabel:`MongoDB connection string (URI)`.
 
-      Connection and SSL details depend on the database type you are connecting to. 
-      In addition to the generic connection properties listed above, you may also need
-      to select a :guilabel:`Identifier` for Oracle and :guilabel:`Authentication`
-      for SQL Server.
+      If you leave any of the form fields for :guilabel:`Database`, :guilabel:`Username`, 
+      or :guilabel:`Password` blank, the values specified in the URI are used.
 
-   If you want to specify the JDBC URI manually, click the 
-   :guilabel:`Enter URI manually` toggle switch on the :guilabel:`Connect SourceDB` 
-   form. For details, see :ref:`rm-relational-database-connection-strings`
-   .
+      Click :guilabel:`Connect`.
 
-#. Click :guilabel:`Connect`. 
+  .. step:: Select your migration options
 
-#. On the :guilabel:`Connect Destination DB` form, input your 
-   :guilabel:`MongoDB connection string (URI)`.
+      On the :guilabel:`Migration Options` form, select your 
+      :guilabel:`Migration Options`:
 
-   If you leave any of the form fields for :guilabel:`Database`, :guilabel:`Username`, 
-   or :guilabel:`Password` blank, the values specified in the URI are used.
+      .. list-table::
+        :header-rows: 1
 
-#. Click :guilabel:`Connect`.
+        * - Migration Option
+          - Description
+        * - Mode 
+          - Defines the type of sync job.
+        * - Drop destination collections before migration
+          - Boolean. Indicates whether Relational Migrator drops a 
+            destination collection before transferring data. 
+        * - Stop after errors 
+          - Integer. Indicates the number of errors after which Relational 
+            Migrator stops the sync job.  
+        * - Verify migrated data 
+          - Boolean. If true, the sync engine checks the migrated data 
+            against the source database. Only supported for snapshot mode.
 
-#. On the :guilabel:`Migration Options` form, select your 
-   :guilabel:`Migration Options`:
+      Once you have established the database connection and specified the 
+      job type on the :guilabel:`Migration Options` form, Relational Migrator
+      conducts various checks to ensure that the database is configured 
+      correctly. If any configuration is missing, a banner appears indicating
```

Comment:
[Suggestion]

"If a configuration is missing, a banner indicates improper database setup and provides a "Generate Script" button for SQL script download."